impl/src/main/java/org/kuali/rice/kew/web/UserLoginFilter.java

```diff
@@ -1,151 +1,157 @@
-
-    public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain) throws IOException, ServletException {
-        if (!(req instanceof HttpServletRequest && res instanceof HttpServletResponse)) {
-            chain.doFilter(req, res);
-            return;
-        }
-
-        LOG.debug("Begin UserLoginFilter...");
-
-        HttpServletRequest request = (HttpServletRequest) req;
-        HttpServletResponse response = (HttpServletResponse) res;
-
-        final UserSession userSession;
-        if (!isUserSessionEstablished(request)) {
-            userSession = login(request);
-            if (userSession != null) {
-                request.getSession().setAttribute(KEWConstants.USER_SESSION_KEY, userSession);
-            }
-        } else {
-            userSession = (UserSession) request.getSession().getAttribute(KEWConstants.USER_SESSION_KEY);
-        }
-
-        if (userSession != null) {
-            // Override the HttpServletRequest with one that provides
-            // our logged-in user. This allows any engine-agnostic webapp code
-            // that may be living in the context to obtain remote user traditionally
-            LOG.debug("Wrapping servlet request: " + userSession.getPrincipalName());
-            request = new HttpServletRequestWrapper(request) {
-                public String getRemoteUser() {
-                    return userSession.getPrincipalName();
-                }
-            };
-        }
-
-        // set up the thread local reference to the current authenticated user
-        // and then forward to next filter in the chain
-        MDC.put("user", userSession.getPrincipalName());
-        try {
-            UserSession.setAuthenticatedUser(userSession);
-            LOG.debug("...end UserLoginFilter.");
-            chain.doFilter(request, response);
-        } finally {
-        	MDC.remove("user");
-            UserSession.setAuthenticatedUser(null);
-        }
-
-    }
-
-    /**
-     * Checks if the user who made the request has a UserSession established
-     *
-     * @param request
-     *            the HTTPServletRequest object passed in
-     * @return true if the user session has been established, false otherwise
-     */
-    public static boolean isUserSessionEstablished(HttpServletRequest request) {
-        return (request.getSession(false) != null && request.getSession(false).getAttribute(KEWConstants.USER_SESSION_KEY) != null);
-    }
-
-    /**
-     * create a UserSession object for the workflow user
-     *
-     * @param request
-     *            the servlet request
-     * @return UserSession object if authentication was successful, null otherwise
-     */
-    protected UserSession login(HttpServletRequest request) {
-        LOG.info("performing user login: ");
-
-        String principalName = null;
-        KimPrincipal principal = null;
-
-        IdentityManagementService idmService = KIMServiceLocator.getIdentityManagementService();

```

Comment:
getAuthenticatedPrincipalName method was removed.   This is the only line that changed in this class.